### PR TITLE
Use sequelize for clients API model

### PR DESCRIPTION
**Pull request by ferjm**

### CODE REVIEW

File: src/routes/clients.js [hunk 1]
```diff
@@ -36,23 +37,31 @@ router.post('/', (req, res) => {
...
-  }).catch(error => {
-    ApiError(res, 500, ERRNO_INTERNAL_ERROR, INTERNAL_ERROR);
-  })
+  db().then(models => {
+    models.Clients.findAll({
+      attributes: ['key', 'name']
+    }).then(clients => {
+      res.status(200).send(clients);
+    }).catch(error => {
+      ApiError(res, 500, ERRNO_INTERNAL_ERROR, INTERNAL_ERROR,
```
Comment (albertopq) [inline src/routes/clients.js]: Do we want to expose sequelize errors?

File: src/models/db.js [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+import config    from '../config';
+import fs        from 'fs';
+import path      from 'path';
+import Sequelize from 'sequelize';
+
+const IDLE           = 0
+const INITIALIZING   = 1;
+const READY          = 2;
+
+const Deferred = function() {
```
Comment (albertopq) [inline src/models/db.js]: Can we name this method ¨Deferred" as well?

File: src/routes/clients.js (hunk 1 above)
Comment (ferjm) [inline src/routes/clients.js]: Yes, they provide a better understanding of the error

Comment (albertopq) [inline src/routes/clients.js]: What I mean is that it could potentially expose internal database information. Should we parse the sequelize response?

Comment (ferjm) [inline src/routes/clients.js]: Ah, yes. That's a good point. I don't think sequelize will dump any private content in this case, but I guess it's better to play safe in this case.

**Pull request merged**